microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,180 @@
+1. Open Microsoft Edge Beta, Dev, or Canary (just go to edge://version and ensure you're on version 105+)
+2. Navigate to URL protocol handler (protocol-handler.glitch.me).
+3. Install the PWA (see docs).
+4. Open DevTools > Application tool.
+5. Select the Manifest dropdown to expand it.
+6. Select the Protocol Handler entry in the dropdown.
+7 With the web+coffee:// protocol selected, enter into the textbox "americano".
+8 Take a screenshot. Draw red highlight boxes over the "Protocol handlers" entry in the Manifest dropdown and around the Protocol Handlers section in the panel. -->
+
+![1](./devtools-105-images/protocol-handler.png)
+
+See also:
+
+* [Test Progressive Web App (PWA) protocol handling](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/progressive-web-apps/protocol-handlers)
+
+<!-- ====================================================================== -->
+
+## Edit and resend network requests more reliably in the Network Console tool
+
+<!-- RANK 3 -->
+<!-- Title: Edit and resend network requests more reliably in the Network Console tool  -->
+<!-- Subtitle: Modify and resend network requests that have been logged in the Network tool with the Network Console tool. -->
+
+Previously in Microsoft Edge, selecting **Edit and resend** for a network request from the Network tool may not have reliably opened the Network Console tool and prepopulated it with the values from the request. 
+
+In Microsoft Edge 105, this issue has been fixed. The **Edit and resend** now prepopulates the Network Console tool with the values for the network request that you are looking to resend. You can continue to modify these values before sending the request from the Network Console tool. Thank you for sending us your feedback about this issue!
+
+Right-click the network request that you want to change and resend, and then select *Edit and Resend*.
+
+[1](./devtools-105-images/edit-and-resend.msft.png
+
+In the Network Console, edit the network request information, and then click the *Send* button.
+[2](./devtools-105-images/networkconsole-edit.msft.png
+
+
+See also:
+* [Compose and send web API requests using the Network Console tool](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/network-console/network-console-tool#starting-from-the-network-tool)
+
+<!-- ====================================================================== -->
+## Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View
+
+<!-- Rank 3 -->
+<!-- Title: Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View -->
+<!-- Subtitle: Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View. -->
+
+Several improvements to [Focus Mode](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode) in existing location controls for DevTools have been made, including a new control for changing the orientation of Quick View.
```

Comment:
Global: add bold to Quick View and Activity Bar per conventions in https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/focus-mode

Reply:
fixed in local and will publish edits